tokens/contracts/access/MintControl.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract MintControl is Initializable, OwnableUpgradeable {
+    mapping (address => bool) private _minters;
+    bool public minterControlEnabled;
+
+    event MinterControlEnable(address indexed sender);
+    event MinterControlDisable(address indexed sender);
```

Comment:
Why send the sender address here?

Reply:
It's a common pattern to send the address of the user doing the admin action (almost "required" if using `AccessControl`), but it can be removed in our case.